pkg/settings/setting.go

```diff
@@ -145,6 +145,7 @@ var (
 	AuthUserSessionTTLMinutes = NewSetting("auth-user-session-ttl-minutes", "960") // 16 hours
 
 	// ConfigMapName name of the configmap that stores rancher configuration information.
+	// Deprecated
```

Comment:
This should never have been used, so fine to remove in 2.8.0: 
```suggestion
	// Deprecated: to be removed in 2.8.0
```